syftlib/src/main/java/org/openmined/syft/proto/SyftTensor.kt

```diff
@@ -168,5 +149,28 @@ data class SyftTensor(
             else -> throw Exception("Invalid Tensor type")
         }
     }
+}
 
-}
\ No newline at end of file
+@ExperimentalUnsignedTypes
+fun SyftProtoTensor.deserialize(): SyftTensor {
```

Comment:
I feel this should not be public, it is a specific type of object instantiation rather than deserialising. If public should be renamed to something like getSyftTensor()

Reply:
As it is, you are right. My intention is to replace the companion object for the extension function.

Reply:
Yeah we can do away with companion deserialize completely

Reply:
Maybe rename it as `SyftProtoTensor.toSyftTensor()`?